docs/deployment-local-cluster.md

```diff
@@ -0,0 +1,46 @@
+# Deployment on Local Cluster
+
+This project allows you to spin up virtualized Kubernets/OpenShift cluster. In
+this guide we will create a local Kubernetes cluster with two nodes and preinstalled
+node dependencies, then we will deploy kubernetes-nmstate from local sources.
+
+If you want to deploy kubernetes-nmstate on your arbitrary cluster, go to
+[deployment on arbitrary cluster guide](deployment-arbitrary-cluster.md).
+
+Start local cluster. If you want to use OpenShift instead of Kubernetes or
+different amount of nodes, check [development guide](developer-guide.md#local-cluster).
```

Comment:
*a* different amount of nodes, check *the* ...

Reply:
Done